src/pages/Autolending/InlineCounter.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<h4>
+	<h4 :class="{'bold' : strong, 'normal': !strong }">
```

Comment:
The inline counter lives in 2 places now. One of them bold, the other regular weight. This new prop determines the css style.